src/SampleWebApp/Models/AuthorDataContext.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using SampleWebApp.Models;
+
+namespace SampleWebApp.Data
+{
+    public class AuthorDataContext : DbContext
```

Comment:
Is it an explicit decision to call these generated classes "Data Context"? Not sure it is a good idea to come up with yet another variation on what to call one of these on an already crowded space.